javascript.md

```diff
@@ -55,7 +60,11 @@ let bar = thing; // throws, undefined
 Rust:
 
 ```rust
-// TODO
+if true {
```

Comment:
Considering the jscs equivalent: the compiler warns you about some style issues. There is also rustfmt ( currently in development )  that formats rust code according to certain rules. IIRC a project called clippy adds more lints to the compiler.